app/build.gradle

```diff
@@ -30,6 +30,16 @@ android {
         buildConfigField "String", "VERSION_NAME_FOR_DISPLAY", "VERSION_NAME"
         resValue "string", "com.crashlytics.android.build_id", "$majorAndMinorVersion.$patchVersionCode"
     }
+
+    signingConfigs {
+        release {
+            storeFile file(envVariable("keystore",true))
```

Comment:
Can you check whether

    storeFile file(envVariable("keystore",true) ?: "DUMMY")

would work?